ush/fv3gfs_driver_grid.sh

```diff
@@ -171,7 +171,7 @@ if [ $gtype = uniform ] || [ $gtype = stretch ] || [ $gtype = nest ];  then
     touch $TMPDIR/orog.file1
     tile=1
     while [ $tile -le $ntiles ]; do
-      echo "$script_dir/fv3gfs_make_orog.sh $res $tile $grid_dir $orog_dir $script_dir $topo $TMPDIR " >>$TMPDIR/orog.file1
+      echo "$script_dir/fv3gfs_make_orog.sh $res $tile $grid_dir $orog_dir $topo $add_lake $TMPDIR " >>$TMPDIR/orog.file1
```

Comment:
The argument list for all calls to this script need to be updated.   When I try to run a regional grid, it fails.